src/web_sys.rs

```diff
@@ -1302,6 +1302,30 @@ impl Context {
             RawRenderingContext::WebGl2(ref gl) => gl.bind_framebuffer(target, Some(framebuffer)),
         }
     }
+
+    // Returns whether the "ldr" profile is support by the `WEBGL_compressed_texture_astc`
+    // extension, with `None` indicating that the extension itself isn't supported.
+    pub fn compressed_texture_astc_get_ldr_profile_support(&self) -> Option<bool> {
```

Comment:
Maybe the name `compressed_texture_astc_supports_ldr_profile` reads a little nicer (same for hdr)?